build-logic/src/main/kotlin/nessie-conventions-root.gradle.kts

```
@@ -35,7 +35,8 @@ plugins {
 loadNessieProjects(rootProject)
 
 val projectName = rootProject.file("ide-name.txt").readText().trim()
-val ideName = "$projectName ${rootProject.version.toString().replace(Regex("^([0-9.]+).*"), "$1")}"
+val ideName =
+  "$projectName ${rootProject.version.toString().replace(Regex("^([0-9.]+).*"), "$1")} [${rootProject.rootDir}]"
```

Would the last path element be sufficient?
```suggestion
  "$projectName } [in ../${rootProject.rootDir.name}]"
```

I'm okay to remove the version

---

Let's leave the version for now since it's used for Eclipse as well, and maybe Eclipse has a better synchronization.